packages/core/src/configuration/ConfigurationHelper.ts

```diff
@@ -0,0 +1,26 @@
+import find from "lodash.find";
+import { AeroGearConfiguration, ServiceConfiguration } from "./";
+
+/**
+ * Helper class to read service configurations from a mobile-services.json file
+ */
+export class ConfigurationHelper {
+
+  protected readonly configurations: ServiceConfiguration[];
+
+  /**
+   * @param config - top level configuration that will be send from server.
+   */
+  constructor(config: AeroGearConfiguration) {
+    this.configurations = config.services || [];
+  }
+
+  /**
+   * Get a service configuration object, provided an existing id is given
+   * @param id string - The id of the service
+   */
+  public getConfig(id: string): ServiceConfiguration | undefined {
+    return find(this.configurations, service => service.id === id);
```

Comment:
just to avoid potential problem of using uppercase/lowercase letters, let's ignore the cases here.

Reply:
I would say the opposite, we should consider that an error IMO. An id should be unique and we should expect our service ids are standardize, that's why we define this `MyService.ID` constant, to avoid case errors. @wtrocki wdty?

Reply:
My thoughts are:

1. there is nothing stop developers from just passing a string. I think it should be allowed.
2. there could be custom services added in the future, and we may not be able to control the id value there. 

I think we should align this with the current naming convention on the server side. Are we ignoring cases on the backend? @maleck13 any idea?

Reply:
This will be only inside of our SDK's so if there is case issue it should fail badly and we should correct that.